eco-conception/le-service-privilegie-la-low-tech.md

```diff
@@ -1 +1,14 @@
-# Le service privilégie la low-tech
\ No newline at end of file
+# Le service privilégie la low-tech
+
+## Description
+
+Le service adopte une approche low-tech pour minimiser l'impact environnemental et maximiser l'efficacité.
+
+## Critères
+
+- [ ] Les solutions low-tech sont privilégiées dans le développement.
+- [ ] L'impact environnemental est régulièrement évalué.
```

Comment:
Est-ce qu'on peut viser directement l'IA aussi ? genre pas mettre de l'ia pour de l'ia ?

Reply:
excellente question, voir est-ce qu'on rajoute carrément des règles IA façon GOV.UK qui ont sorti un [playbook dédié à l'IA](https://www.gov.uk/government/publications/ai-playbook-for-the-uk-government) ?

Reply:
J'aimerai trop, mais je me sens pas les épaules (par contre, je vais lire le playbook)

Reply:
Parlons en ici : https://github.com/betagouv/standards/issues/12